.circleci/config.yml

```diff
@@ -314,15 +314,63 @@ jobs:
       - run:
           name: Ensure non-editable mode works
           command: |
+              cd habitat-lab
               export PATH=$HOME/miniconda/bin:/usr/local/cuda/bin:$PATH
-              . activate habitat; cd habitat-lab
-              python -m pip install habitat-lab/
-              python -c "import habitat"
-              python -m pip install habitat-baselines/
-              python -c "import habitat_baselines"
+              conda create -y -n non-editable-install python=3.9
+              . activate non-editable-install
+              # conda install -c aihabitat-nightly habitat-sim
+              pip install habitat-lab/
+              # python -c 'import habitat; print("habitat version:", habitat.__version__)'
+              pip install habitat-baselines/
+              # python -c 'import habitat_baselines; print("habitat_baselines version:", habitat_baselines.__version__)'
```

Comment:
This was added in [wait for habitat-sim build with python 3.9 available on conda](https://github.com/facebookresearch/habitat-lab/pull/1192/commits/a9059599f7aaccd2c5af950469180a1a3084626c) commit. As you may see in line 321 habitat-sim conda installation is also commented too, because due to python version upgrade it is not installable via conda yet. But it is installable from source, so as a workaround if habitat-sim python upgrade is not ready before the release I can install habitat sim from source and uncomment lab/baselines installation check.